src/sentry/index.ts

```diff
@@ -0,0 +1,20 @@
+import * as Sentry from "@sentry/browser";
+
+export const initializeSentry = () => {
+  const sentryDSN = window.REEARTH_CONFIG?.sentryDsn;
+  const sentryEnv = window.REEARTH_CONFIG?.sentryEnv;
+  if (sentryDSN) {
+    Sentry.init({
+      dsn: sentryDSN,
+      environment: sentryEnv,
+    });
+  }
+};
+
+export const reportError = (error: string) => {
+  Sentry.captureException(error);
```

Comment:
Have you verified that this function does not generate an error even if sentry is not initialized?

Reply:
I've tested it if it works without initilization. 👍